asciidoc/client-applications.adoc

```diff
@@ -63,87 +57,67 @@ include::{javascript-examples}/examples.test.js[tags=driver-lifecycle]
 ----
 ======
 
-[.include-with-python]
-======
-[source, python]
-----
-include::{python-examples}/driver_lifecycle_example.py[tags=driver-lifecycle]
-----
-======
+// [.include-with-python]
+// ======
+// [source, python]
+// ----
+// include::{python-examples}/driver_lifecycle_example.py[tags=driver-lifecycle]
+// ----
+// ======
 ====
 
 
 [[driver-connection-uris]]
 == Connection URIs
 
-A connection URI identifies a graph database and how to connect to it.
-The official Neo4j drivers currently support the following URI schemes and driver object types:
+Connection URIs are typically formed according to the following pattern:
 
-.Available URI schemes
-[options="header"]
-|===
-| URI Scheme     | Driver object type
-| bolt           | Direct driver
-| bolt+routing   | Routing driver
-|===
+----
+neo4j://<HOST>:<PORT>[?<ROUTING_CONTEXT>]
+----
 
+This targets a routed Neo4j service that may be fulfilled by either a cluster or a single instance.
+The HOST and PORT values contain a logical hostname and port number targeting the entry point to the Neo4j service (e.g. neo4j://graph.example.com:7687).
+In a clustered environment, this will resolve to one of more of the core members; for standalone installations, this will simply point to that server address.
+The ROUTING_CONTEXT option allows for customization of the routing table and is discussed in more detail in <<driver-routing-context>>.
 
-=== Direct drivers (bolt)
+An alternative URI form, using the bolt URI scheme (e.g. `bolt://graph.example.com:7687`), can be used when a single point-to-point connection is required.
+This variant is useful for the subset client applications (such as admin tooling) that need to be aware of individual servers, as opposed to those which require a highly available database service.
```

Comment:
It is impossible to target a specific machine with the `neo4j` URI scheme, since that targets an overall service, which is usually comprised of multiple machines. The `bolt` URI scheme denotes a point-to-point connection for when an actual server needs to be targeted, regardless of any cluster that server may also be a part of.

In terms of formatting, we need to draw a distinction between Bolt (the protocol) and `bolt` (the URI scheme). Similar to the distinction between Neo4j (the product) and `neo4j` (the URI scheme).

Reply:
changed to "...using the `bolt` URI scheme..."